src/index.html

```diff
@@ -5,8 +5,361 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+
+  <body class="body">
+    <header class="header">
+      <a class="header__elem" href="#home">
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="samsung"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
```suggestion
            </a>
          </li>
          
          <li class="nav__item">
            <a
              href="samsung"
              class="nav__link"
            >
              Samsung
            </a>
          </li>
          
          <li class="nav__item">
```

Reply:
Pay attention to the checklist, fix code style all over the page <img width="998" alt="Screenshot 2022-11-22 at 19 02 58" src="https://user-images.githubusercontent.com/104769168/203376385-e9720fa0-216d-417a-ad07-724f9ed1bc5e.png">